Makefile

```diff
@@ -0,0 +1,11 @@
+.PHONY: list
+
+list:
+	@LC_ALL=C $(MAKE) -pRrq -f $(lastword $(MAKEFILE_LIST)) : 2>/dev/null | awk -v RS= -F: '/^# File/,/^# Finished Make data base/ {if ($$1 !~ "^[#.]") {print $$1}}' | sort | egrep -v -e '^[^[:alnum:]]' -e '^$@$$'
+
+py:
```

Comment:
This target shouldn't be called py, it should be called "eventio_kaitai_parser.py", i.e. the output name of the workflow